kubernetes/schema_container.go

```diff
@@ -587,6 +587,19 @@ func containerFields(isUpdatable bool) map[string]*schema.Schema {
 				Schema: resourcesFieldV1(isUpdatable),
 			},
 		},
+		"restart_policy": {
```

Comment:
From first glance be sure to include a test case for this new field. You also want to make sure to add a changelog-entry, this is mentioned in [CHANGELOG_GUIDE](https://github.com/hashicorp/terraform-provider-kubernetes/blob/main/CHANGELOG_GUIDE.md#how-to-use)

The changelog entry and new test should be added into the description of the PR also.

Reply:
Although you added the changelog you are still missing a testcase that adds this. You would be adding this inside https://github.com/hashicorp/terraform-provider-kubernetes/blob/bff3ae2a2cc4552fc8da1ba57720f652f8b1beaa/kubernetes/resource_kubernetes_deployment_v1_test.go#L90-L196

as well as https://github.com/hashicorp/terraform-provider-kubernetes/blob/bff3ae2a2cc4552fc8da1ba57720f652f8b1beaa/kubernetes/resource_kubernetes_pod_v1_test.go#L143-L187

Refer to the contribution guideline to learn how to run tests locally on your machine